Sources/NIOWebSocket/WebSocketProtocolErrorHandler.swift

```diff
@@ -22,7 +22,12 @@ public final class WebSocketProtocolErrorHandler: ChannelInboundHandler {
     public typealias InboundIn = Never
     public typealias OutboundOut = WebSocketFrame
 
-    public init() {}
+    /// Indicate that this `ChannelHandeler` is used by a WebSocket server or client. Default is true.
+    public private(set) var isServer: Bool
```

Comment:
Let's avoid making anything `public` we don't have to. I can't think of many uses for having this be public so we should probably avoid it.